src/logic/embedding.lean

```diff
@@ -194,6 +194,10 @@ def subtype {α} (p : α → Prop) : subtype p ↪ α :=
 
 @[simp] lemma coe_subtype {α} (p : α → Prop) : ⇑(subtype p) = coe := rfl
 
+/-- `quotient.out` as an embedding. -/
+@[simps] noncomputable def quotient_out (α) [s : setoid α] : quotient s ↪ α :=
```

Comment:
What does `simps` generate here?

Reply:
I think you want the lemma ` coe_quotient_out : ⇑(quotient_out α) = quotient.out`, and should add it manually if that's not what you get.

Reply:
It only gives the applied version; I'll prove this manually instead.